kornia/enhance/rgb_shift.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+
+def shift_rgb(image, r_shift, g_shift, b_shift):
+    """
+    Shift each image's channel by either r_shift for red, g_shift for green and b_shift for blue channels.
+    """
+
+    if not isinstance(image, torch.Tensor):
```

Comment:
I suggest To use the api below
https://github.com/kornia/kornia/blob/master/kornia/testing/__init__.py#L208

Also add the params and shapes the docstring